### DISCUSSION TITLE
Query Execution In Progress PopUp

### DISCUSSION THREAD

**Discussion by VishwamberShetty** [superfell/SoqlX#165] [Ideas]
1. Write a Query

2. Click the Button Query to Execute it

3. PopUp Indicating the Query Execution in Progress (Stays till the Execution is Completed)

4. Once the Execution is Completed, the PopUp will display the Total Records Fetched

5. It will also Ask if the User wants to Display all the Records on the Query Result Grid

6. If Selected **Yes**, then All the Resultant Records will be displayed on the Grid

7. If Selected **No**, then Only the Record Count to Display on the Resultant Grid will be decided by **Default Result Display Record Count** Set in Setup will be considered.

**Comment by superfell** [comment]
What problem are you trying to solve by adding these extra popups & buttons?
